code-examples/php/usage-examples/find.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once __DIR__ . "/vendor/autoload.php";
+
+$uri = "mongodb://localhost:27017";
+
+$client = new MongoDB\Client($uri);
+
+try {
+    $collection = $client->sample_mflix->movies;
```

Comment:
Constructing a collection doesn't need to happen within a `try/catch` (assuming we even need a `try/catch`). Making this file consistent with the others would avoid this entirely.